app/src/main/java/com/android/sample/receivers/NotificationReceiver.kt

```diff
@@ -0,0 +1,29 @@
+package com.android.sample.receivers
+
+import android.app.NotificationManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import com.android.sample.R
+
+class NotificationReceiver : BroadcastReceiver() {
+  override fun onReceive(context: Context, intent: Intent) {
```

Comment:
If we want to create new types of notifications, should we use this onReceive or create a new function ? One type of notifcation (24hour reminder) is hard-coded here and it would be great to modularise it.

Reply:
Answered this in a previous comment, but it's all good, I will create a new class: ActivityDeletionNotificationManager()